src/data/faq_de.json

```diff
@@ -52,13 +52,11 @@
-                            "Hinweise:<ul><li>Die Fehlerberichte werden nach 14 Tagen automatisch gelöscht</li><li>Berichte, die unaufgefordert an den Server übertragen werden, können durch die Entwickler nicht analysiert werden</li><li>Für die Fehlerberichte gibt es keinen direkten Rückkanal in der Corona-Warn-App. Der technische Support kann nur über einen der oben genannten Wege mit Ihnen kommunizieren</li></ul>",
-                            "Weitere Infos sowie Screenshots finden Sie im Blog-Artikel <a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>Corona-Warn-App Version 2.2 ermöglicht die Erstellung von Schnelltest-Profil und Fehlerbericht</a>."
+                            "Mit Version 2.2 bietet die Corona-Warn-App eine Funktion zur technischen Problem-Analyse. Fehlerberichte sollen den Entwicklern helfen, schneller eine Ursachen-Analyse zu erstellen und Fehler effizienter zu beseitigen.<br><b>Wichtig:</b> Senden Sie bitte nur einen Fehlerbericht, falls dieser vom technischen Support angefragt wird.",
+                            "Folgen Sie diesen Schritten, um einen Fehlerbericht zu erstellen:",
+                            "<ol><li>Öffnen Sie auf dem Hauptbildschirm der Corona-Warn-App das Dreipunktmenü in der oberen rechte Ecke und wählen Sie 'App-Informationen' -> 'Fehlerberichte'.</li><li>Wählen Sie 'Start' am unteren Bildschirmrand. Die Fehleraufzeichnung wird dadurch im Hintergrund aktiviert.</li><li>Versuchen Sie das fehlerhafte Verhalten zu reproduzieren bzw. die Fehlermeldung zu erzeugen.</li><li>Ist das gelungen, senden Sie den Fehlerbericht an den Server der Corona-Warn-App.</li><ol><li>Rufen Sie das 'Fehlerberichte' Menü erneut auf.</li><li>Wählen Sie 'Fehlerbericht senden' am unteren Bildschirmrand. Diese Aktion sendet die Aufzeichnungsdaten an den Server der Corona-Warn-App. Die Daten werden dabei anonymisiert und verschlüsselt.</li><li>Stoppen Sie die Fehleranalyse</li></ol><li><b>Teilen Sie die ID des Fehlerberichts mit dem technischen Support.</b> Sie finden die ID ihres Fehlerberichts auf dem 'Fehlerberichte' Bildschirm. Scrollen Sie bis nach unten und klicken Sie auf 'ID-Historie'. Am besten teilen Sie die ID an dem Ort mit, wo Sie vom technischen Support angefragt wurde.</ol>",

```

Comment:
The "Hauptbildschirm" is labelled "Startseite" with the house symbol with locale set to German. See https://www.coronawarn.app/de/screenshots/. Maybe also use that name here?

Reply:
fixed